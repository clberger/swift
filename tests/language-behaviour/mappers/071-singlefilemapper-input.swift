type messagefile;

app (messagefile o) write(messagefile i) {
  echo @filename(i) stdout=@filename(o);
}

messagefile infile <"071-singlefilemapper-input.in">;
messagefile outfile <"071-singlefilemapper-input.out">;

outfile = write(infile);

