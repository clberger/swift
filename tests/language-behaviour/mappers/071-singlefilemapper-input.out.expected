071-singlefilemapper-input.in
